odl_video/settings.py

```diff
@@ -505,9 +515,18 @@ def get_all_config_keys():
     'VIDEO_S3_THUMBNAIL_BUCKET',
     'VIDEO_S3_SUBTITLE_BUCKET',
     'VIDEO_S3_WATCH_BUCKET',
-    'ENABLE_VIDEO_PERMISSIONS'
+    'ENABLE_VIDEO_PERMISSIONS',
 ]
 
+if ENABLE_VIDEO_PERMISSIONS:
+    MANDATORY_SETTINGS += [
```

Comment:
Can you add a test that these values are used when constructing `YoutubeApi`?

Reply:
OK